### PR TITLE
add lastpass attachment export script

### PR DESCRIPTION
**Pull request by mindrunner**
https://github.com/lastpass/lastpass-cli/issues/412

### CODE REVIEW

File: contrib/lpass-att-export.sh [hunk 1]
```diff
@@ -2,35 +2,84 @@
 ##
 ## Usage: lpass-att-export.sh
 ##
-##  - login via lpass login
-##  - run ./lpass-att-export.sh
-##  - check subfolder 'lpass-export'
 ##
 
+set -x
```
Comment (waterkip) [inline contrib/lpass-att-export.sh]: I think this is a debug artifact hanging around

File: contrib/lpass-att-export.sh [hunk 2]
```diff
@@ -2,35 +2,84 @@
...
+  attcount=$(echo "${show}" | grep -c "att-")
+  path=$(lpass show --format="%/as%/ag%an" ${id} | uniq | tail -1)
+
 
-  path=`lpass show ${id} | sed "1q;d" | awk '{$NF=""; print $0}' | awk '{$NF=""; print $0}' | awk '{$1=$1};1'`
-  attcount=`lpass show ${id} | grep att- | wc -l`
 
   until [  ${attcount} -lt 1 ]; do
+
     att=`lpass show ${id} | grep att- | sed "${attcount}q;d" | tr -d :`
```
Comment (waterkip) [inline contrib/lpass-att-export.sh]: echo $show | grep att ? :)

File: contrib/lpass-att-export.sh (hunk 1 above)
Comment (mindrunner) [reply contrib/lpass-att-export.sh]: Done!

File: contrib/lpass-att-export.sh (hunk 2 above)
Comment (mindrunner) [reply contrib/lpass-att-export.sh]: I tried, but had some prolems with newlines and ran out of time, so I reverted back to what I had before. Works better like this. Might have time to look into next week.

**Comment by mindrunner** [comment]
Are you waiting for me fixing more stuff?

**Comment by alexkuc** [comment]
Any chance to get it merged? Looks really important to those who want to keep a local backup copy of LastPass's data.

**Comment by mindrunner** [comment]
+1
Whats missing to get this done?

**Comment by GPaulovics** [comment]
Hey @mindrunner, sorry for the long waiting.
I will try to keep an eye on the project from now on.

The problem is that you have a commit that is not signed off, and I've got the instructions not to merge in commits that are not signed off. 

So first that should be solved. After that I would be happy to merge your pull request.
And Thank You for your contribution.

**Comment by mindrunner** [comment]
Will have access to my computer earliest in April. Signing off the commits then!

**Comment by mindrunner** [comment]
Should be fine now!

**Comment by GPaulovics** [comment]
Yepp, Thank You @mindrunner for the contribution and @waterkip for the review.

**Pull request merged**